AstSemantics.md

```diff
@@ -375,7 +375,7 @@ results into the result type.
   * `int32.sub`: sign-agnostic subtraction
   * `int32.mul`: sign-agnostic multiplication (lower 32-bits)
   * `int32.div_s`: signed division (result is truncated toward zero)
-  * `int32.div_u`: unsigned division
+  * `int32.div_u`: unsigned division (result is floored)
```

Comment:
They're equivalent for unsigned types, because there are no negative values. Floor is a common term though; I'll add a link.